code/modules/client/preference/preferences.dm

```diff
@@ -819,13 +819,18 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 	var/HTML = "<body>"
 	HTML += "<tt><center>"
 
-	HTML += ShowDisabilityState(user,DISABILITY_FLAG_NEARSIGHTED,"Needs Glasses")
+	HTML += ShowDisabilityState(user,DISABILITY_FLAG_NEARSIGHTED,"Needs glasses")
 	HTML += ShowDisabilityState(user,DISABILITY_FLAG_FAT,"Obese")
 	HTML += ShowDisabilityState(user,DISABILITY_FLAG_EPILEPTIC,"Seizures")
 	HTML += ShowDisabilityState(user,DISABILITY_FLAG_DEAF,"Deaf")
 	HTML += ShowDisabilityState(user,DISABILITY_FLAG_BLIND,"Blind")
 	HTML += ShowDisabilityState(user,DISABILITY_FLAG_COLOURBLIND,"Colourblind")
 	HTML += ShowDisabilityState(user,DISABILITY_FLAG_MUTE,"Mute")
+	HTML += ShowDisabilityState(user,DISABILITY_FLAG_TOURETTES,"Tourettes syndrome") // this will / can not be abused. It also SEVERELY stuns. It's just for fun.
+	HTML += ShowDisabilityState(user,DISABILITY_FLAG_NERVOUS,"Stutter")
+	HTML += ShowDisabilityState(user,DISABILITY_FLAG_SWEDISH,"Swedish accent")
+	HTML += ShowDisabilityState(user,DISABILITY_FLAG_LISP,"Lisp")
+	HTML += ShowDisabilityState(user,DISABILITY_FLAG_DIZZY,"Suffers from dizziness")
```

Comment:
Wouldn't it just be better to have "Dizziness"? "Suffers from dizziness" seems a bit excessive since it's not written like that elsewhere.